scripts/build.js

```diff
@@ -97,8 +104,8 @@ if (!HAS_WATCH_FLAG) {
     console.log('Remove `export {}` from CJS files')
     for (const pkg of ['webdriver', 'devtools', 'webdriverio']) {
         const filePath = path.join(__dirname, '..', 'packages', pkg, 'build', 'cjs', 'index.js')
-        const fileContent = await fs.readFileSync(filePath, 'utf8')
-        await fs.writeFileSync(filePath, fileContent.toString().replace('export {};', ''), 'utf8')
+        const fileContent = fs.readFileSync(filePath, 'utf8')
+        fs.writeFileSync(filePath, fileContent.toString().replace('export {};', ''), 'utf8')
```

Comment:
Let's keep this and import `node:fs/promises` instead.

```suggestion
        const fileContent = await fs.readFile(filePath, 'utf8')
        await fs.writeFile(filePath, fileContent.toString().replace('export {};', ''), 'utf8')
```